alderaan/pipeline-scripts/prometheus.groovy

```diff
@@ -0,0 +1,104 @@
+			def JUMP_HOST = prometheus_properties['JUMP_HOST']
+			def USER = prometheus_properties['USER']
+			def CLEAR_RESULTS = prometheus_properties['CLEAR_RESULTS']
+			def MOVE_RESULTS = prometheus_properties['MOVE_RESULTS']
+			def USE_PROXY = prometheus_properties['USE_PROXY']
+			def CONTAINERIZED = prometheus_properties['CONTAINERIZED']
+			def PROXY_USER = prometheus_properties['PROXY_USER']
+			def PROXY_HOST = prometheus_properties['PROXY_HOST']
+			def GITHUB_TOKEN = prometheus_properties['GITHUB_TOKEN']
+			def PERF_REPO = prometheus_properties['PERF_REPO']
+			def PBENCH_SERVER = prometheus_properties['PBENCH_SERVER']
+			def SCALE_CI_RESULTS_TOKEN = prometheus_properties['SCALE_CI_RESULTS_TOKEN']
+			def JOB = prometheus_properties['JOB']
+			def REFRESH_INTERVAL = prometheus_properties['REFRESH_INTERVAL']
+			def CONCURRENCY = prometheus_properties['CONCURRENCY']
+			def GRAPH_PERIOD = prometheus_properties['GRAPH_PERIOD']
+			def DURATION = prometheus_properties['DURATION']
+
+			println "----------USER DEFINED OPTIONS-------------------"
+			println "-------------------------------------------------"
+			println "-------------------------------------------------"
+			println "JUMP_HOST: '${JUMP_HOST}'"
+			println "USER: '${USER}'"
+			println "CLEAR_RESULTS: '${CLEAR_RESULTS}'"
+			println "MOVE_RESULTS: '${MOVE_RESULTS}'"
+			println "USE_PROXY: '${USE_PROXY}'"
+			println "CONTAINERIZED: '${CONTAINERIZED}'"
+			println "PROXY_USER: '${PROXY_USER}'"
+			println "PROXY_HOST: '${PROXY_HOST}'"
+			println "GITHUB_TOKEN: '${GITHUB_TOKEN}'"
+			println "PERF_REPO: '${PERF_REPO}'"
+			println "PBENCH_SERVER: '${PBENCH_SERVER}'"
+			println "SCALE_CI_RESULTS_TOKEN: '${SCALE_CI_RESULTS_TOKEN}'"
+			println "JOB: '${JOB}'"
+			println "REFRESH_INTERVAL: '${REFRESH_INTERVAL}'"
+			println "CONCURRENCY: '${CONCURRENCY}'"
+			println "GRAPH_PERIOD: '${GRAPH_PERIOD}'"
+			println "DURATION: '${DURATION}'"
+			println "-------------------------------------------------"
+			println "-------------------------------------------------"
+
+			// copy the parameters file to jump host
+			sh "git clone https://${token}@${repo} ${WORKSPACE}/perf-dept && chmod 600 ${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf"
+			sh "scp -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ${WORKSPACE}/perf-dept/ssh_keys/id_rsa_perf ${property_file_name} root@${jump_host}:/root/properties"
+
+			try {
+				prometheus_build = build job: '${JOB}',
+				parameters: [
+						[$class: 'StringParameterValue', name: 'JUMP_HOST', value: JUMP_HOST ],
+						[$class: 'StringParameterValue', name: 'USER', value: USER ],
+						[$class: 'StringParameterValue', name: 'CLEAR_RESULTS', value: CLEAR_RESULTS ],
+						[$class: 'StringParameterValue', name: 'MOVE_RESULTS', value: MOVE_RESULTS ],
+						[$class: 'StringParameterValue', name: 'USE_PROXY', value: USE_PROXY ],
+						[$class: 'StringParameterValue', name: 'CONTAINERIZED', value: CONTAINERIZED ]]
```

Comment:
The parameters are comma separated values.

Reply:
sure, to who?